llvm/test/CodeGen/X86/O3-pipeline.ll

```diff
@@ -138,9 +138,11 @@
 ; CHECK-NEXT:       Machine Loop Invariant Code Motion
 ; CHECK-NEXT:       Bundle Machine CFG Edges
 ; CHECK-NEXT:       X86 FP Stackifier
+; CHECK-NEXT:       MachineDominator Tree Construction
```

Comment:
This is moved from a couple of lines down, so I don't think something like de92dc2850c17259090ccf644b2f2375ab8e1664 is needed.